### DISCUSSION TITLE
How do you stylize react-easy-edit's date? Or does anyone have an example of using react-datepicker with react-easy-edit?

### DISCUSSION THREAD

**Discussion by niikkiin** [giorgosart/react-easy-edit#136] [Q&A]
Hi! First of allI love using this library as it gives so much ease to the user. 
However I am having difficulty stylizing the date react-easy-edit. Does anyone of you have any idea how I can style it? I also tried using react-datepicker on the `editComponent` but I am having difficulty with it as it always gives me the `data.clone` error. Thanks everyone! 

![image](https://user-images.githubusercontent.com/26428605/138788098-4a2582e4-1de1-417d-8f9b-ba38eaaca519.png)

**Comment by giorgosart** [comment]
Hey @niikkiin can you please provide me with a codepen that has two `EasyEdit` components in it? One with the default date field and another one with the react-datepicker?
